webapp/cas-server-webapp-config/src/main/java/org/apereo/cas/web/security/authentication/MonitorEndpointLdapAuthenticationProvider.java

```diff
@@ -43,10 +43,24 @@
  * @since 5.1.0
  */
 @Slf4j
-@RequiredArgsConstructor
 public class MonitorEndpointLdapAuthenticationProvider implements AuthenticationProvider {
     private final MonitorProperties.Endpoints.LdapSecurity ldapProperties;
     private final SecurityProperties securityProperties;
+    private ConnectionFactory connectionFactory;
+    private Authenticator authenticator;
+
+    public MonitorEndpointLdapAuthenticationProvider(final MonitorProperties.Endpoints.LdapSecurity ldapProperties,
+                                                     final SecurityProperties securityProperties) {
+        this.ldapProperties = ldapProperties;
+        this.securityProperties = securityProperties;
+        this.connectionFactory = LdapUtils.newLdaptiveConnectionFactory(this.ldapProperties);
+        this.authenticator = LdapUtils.newLdaptiveAuthenticator(this.ldapProperties);
+    }
+
+    public void destroy() {
```

Comment:
Missing `@Override`?

Reply:
This particular class is not a bean so I am not declaring it is a `DisposableBean`.